src/pathfinding_service/service.py

```diff
@@ -293,13 +305,6 @@ def _validate_pfs_update(self, message: PFSCapacityUpdate) -> TokenNetwork:
         if token_network is None:
```

Comment:
I can't comment in the signature, but could you also update the naming there? `_validate_pfs_update` -> `_validate_pfs_capacity_update`?

Reply:
Good point! Consistent renaming is harder than it sounds.